src/index.ts

```diff
@@ -12,37 +12,72 @@ import 'css/editor.scss';
 import IPPSettings from './PPSettings.interface';
 declare const PP_SETTINGS: IPPSettings;
 
+// Optional external hardcoded annotations
+import input_annotations from '../config/annotations.json';
+
 console.log('Przypis script working!');
 
-const isBrowser = typeof window !== 'undefined';
-if (isBrowser) {
+function startApp() {
   const annotatorApp = new App();
   annotatorApp.include(PPUI);
   annotatorApp.include(DebugStorage);
   annotatorApp.start();
+  return annotatorApp;
+}
+
+function attachMockViewer() {
+  // IMPORTANT: very implementation dependent; whenever ViewerWidget changes break it, consider simply dropping it
+  const viewer = new ViewerWidget({});
+  viewer.attach();
+
+  const annotation = new AnnotationViewModel({});
+  // mock data to display
+  annotation.comment = 'Testowy komentarz '.repeat(10);
+  annotation.referenceLinkTitle = 'Strona organizacji XYZ '.repeat(3);
+  const position = {
+    top: 500,
+    left: 200,
+  };
+
+  viewer.load([annotation], position);
+  // make the window visible
+  viewer.element.removeClass(ViewerWidget.classes.hide);
+  // turn off the mouseleave listeners so that windows does not disappear on cursor action
+  viewer.element.off('mouseleave.' + ViewerWidget.nameSpace);
+}
+
+function attachAnnotationsFromFile(annotatorApp: App) {
+  // Reassign annotations' ids starting from 1000
+  // so they don't collide with the ids of the annotations created through the application
+  let id = 1000;
+  for (const annotation of input_annotations) {
+    if (annotation.url === window.location.href) {
+      annotation.id = id;
+      annotatorApp.annotations.create(annotation);
+      id += 1;
```

Comment:
maybe instead of overwriting data here, provide some very big random ids in the `json` file?

Reply:
The point is to let anybody add freely their own annotation, without having to worry about ids (and manually generate a random id).

This is a short-lived solution and behaves nicely